dotnet/src/webdriver/ISupportsPrint.cs

```diff
@@ -27,8 +31,9 @@ public interface ISupportsPrint
         /// <summary>
         /// Gets a <see cref="PrintDocument"/> object representing a PDF-formatted print representation of the page.
         /// </summary>
-        /// <param name="options">A <see cref="PrintOptions"/> object describing the options of the printed document.</param>
+        /// <param name="printOptions">A <see cref="PrintOptions"/> object describing the options of the printed document.</param>
         /// <returns>The <see cref="PrintDocument"/> object containing the PDF-formatted print representation of the page.</returns>
-        PrintDocument Print(PrintOptions options);
+        /// <exception cref="ArgumentNullException">If <paramref name="printOptions"/> is <see langword="null"/>.</exception>
+        PrintDocument Print(PrintOptions printOptions);
```

Comment:
Technically breaking change here, the `WebDriver.Print` method and the `ISupportsPrint.Print` methods have different parameter names. It's best practice to align them (there's an analyzer we could turn on) and the `ArgumentNullException` thrown will align to the runtime one.

I figure more people are calling this method directly on `WebDriver` rather than this interface, so I aligned the name with that one. However, if someone is calling `supportsPrint.Print(options: myOptions);` then they will be broken (very simple and obvious fix on their end).

Reply:
Reverted